pyccel/codegen/compiling/compilers.py

```diff
@@ -63,7 +68,7 @@ class Compiler:
                Indicates whether we are compiling in debug mode.
     """
     __slots__ = ('_debug','_info')
-    _acceptable_bin_paths = get_condaless_search_path()
+    acceptable_bin_paths = ()
```

Comment:
I think it would be safer to set the default value to `None`. That way an error will be raised if we change the code in a way that tries to use the class without having first set this property